tests/dataset_runner.py

```diff
@@ -149,28 +166,82 @@ def complete_submission(self):
             raise RuntimeError(f"PUT {submit_url} returned {response.status_code}: {response.content}")
         Progress.report("  done.\n")
 
-    def wait_for_primary_and_results_bundles(self):
+    def wait_for_primary_bundles(self):
         """
         We used to wait for all primary bundles to be created before starting to look for results.
         It appears that with large submissions, results can start to appear before bundle export is finished,
         so we now monitor both kinds of bundles simultaneously.
```

Comment:
What do you think about simultaneously monitoring the primary bundles, workflow status and results bundles? I think that would give us a better idea of what is going on during the test vs. sequential monitoring.